src/ophys_etl/pipelines/dff.py

```diff
@@ -94,6 +97,10 @@ class DffJobSchema(ArgSchema):
                      "+1 if the result is even, since the median filter "
                      "length must be odd).")
     )
+    n_parallel_workers = fields.Int(
+        required=False,
+        default=1,
```

Comment:
Should the default be to learn how many cpus you have access to, so we don't have to change the LIMS strategy to add `n_parallel_workers` to the input.json?

Reply:
in the LIMS UI, we can set `-l nodes=1:ppn=<N>` and then in the args do `--n_parallel_workers <N>` to match them.

Reply:
Right. I forgot that argschema lets you override with command line arguments.